master/buildbot/status/web/builder.py

```diff
@@ -41,9 +41,17 @@ def force(self, req, builderNames):
         owner = self.getAuthz(req).getUsernameFull(req)
         schedulername = req.args.get("forcescheduler", ["<unknown>"])[0]
         if schedulername == "<unknown>":
-            defer.returnValue((path_to_builder(req, self.builder_status),
-                               "forcescheduler arg not found"))
-            return
+            try:
+                for sch in master.allSchedulers():
+                    if isinstance(sch, ForceScheduler) and builderNames[0] in sch.builderNames:
```

Comment:
Can this function even be called with multiple builderNames? I couldn't find an instance of that.